### PR TITLE
Supporting new did-resolver semantics

### PR DESCRIPTION
**Pull request by liamzebedee**
Tested with 1.0.0.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "@babel/runtime": "^7.1.2",
     "did-resolver": "0.0.6",
```
Comment (oed) [inline package.json]: This could be removed now right?

File: package.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
...
+    "ipfs-http-client": "^35.1.0",
```
Comment (oed) [inline package.json]: Why was this added?

**Comment by msterle** [comment]
duplicate of #17

**Comment by liamzebedee** [comment]
Oops, thanks for updating. Forgot to respond here, @oed.
